issue_template.md

```diff
@@ -0,0 +1,19 @@
+**Have a question or install problem?** Please use [Our Forum](https://community.gravitational.com).
+
+Otherwise, tell us:
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
```

Comment:
TBH, this "as minimally and precisely as possible" phrase always bothered me for whatever reason :) I would just say "Steps to reproduce".